unit/testing-utils/require_parse_tree.cpp

```diff
@@ -12,44 +12,42 @@
 /// lambda_method_handle_map with a matching descriptor. Will fail if no
 /// matching lambda entry found.
 /// \param parsed_class: the class to inspect
-/// \param descriptor: the descriptor the lambda method should have
-/// \param entry_index: the number to skip (i.e. if multiple entries with the
-/// same descriptor
+/// \param lambda_method_ref: the reference/descriptor of the lambda method
```

Comment:
Side comment: I think we need a coding guideline on whether there should be a colon after the param name or not (both versions co-exist in this PR). I see the advantage of using a colon to improve readability in the source code, however, doxygen lays it out as follows: http://cprover.diffblue.com/invariant_8h.html#a66872166e18e5b1c3a087f532078b803
Ie, it seems that the colon is part of the description, which may look a bit weird.

Reply:
Good point, I created a PR to propose this in our coding standard https://github.com/diffblue/cbmc/pull/1959